runtime/integration-tests/src/runtime_apis/rewards.rs

```diff
@@ -14,9 +14,10 @@ use cfg_primitives::{AccountId, CFG};
 use cfg_traits::rewards::{AccountRewards, CurrencyGroupChange, DistributedRewards};
 use development_runtime::apis::RewardsApi;
 use frame_support::assert_ok;
+use sp_api::ApiRef;
 use sp_core::{sr25519, Pair};
 use sp_runtime::traits::IdentifyAccount;
-use tokio::runtime::Handle;
+use tokio::runtime::{Handle, Runtime};
```

Comment:
Why is `Runtime` needed here?

Reply:
Good question. Actually no idea how it got added here. Maybe because of the `merge` with the latest `main`. Removed it and all seems fine.